extensions/screen/internal.m

```diff
@@ -447,14 +447,14 @@ static int screen_gammaSet(lua_State* L) {
     lua_getfield(L, 2, "blue");
     whitePoint[2] = lua_tonumber(L, -1);
     lua_pop(L, 1);
-
+    /*
```

Comment:
So these two conditional overrides serve as crude protection against the gamma values being set such that nothing can be seen on the screen anymore. Rather than just disable them, perhaps we can figure out a way to keep the ability to do an inverted mode, without losing the "oh damn I can't see my display anymore" protection? (initial, untested thought, we could check if the white/black points are equal)